articles/tokens/concepts/refresh-tokens.md

```diff
@@ -11,11 +11,11 @@ useCase:
 
 Auth0 issues an <dfn data-key="access-token">[Access Token](/tokens/concepts/access-tokens)</dfn> or an [ID Token](/tokens/concepts/id-tokens) in response to an [authentication request](/api-auth). You can use Access Tokens to make authenticated calls to a secured API, while the ID Token contains user profile attributes represented in the form of *claims*. Both are [JWTs](/tokens/concepts/jwts) and therefore have expiration dates indicated using the `exp` claim, as well as security measures, like signatures. Typically, a user needs a new Access Token when gaining access to a resource for the first time, or after the previous Access Token granted to them expires.
 
-A Refresh Token is a special kind of token that can be used to obtain a renewed access token. You are able to request new access tokens until the Refresh Token is blacklisted. It’s important that refresh tokens are [stored securely](/tokens/guides/store-tokens) by the application because they essentially allow a user to remain authenticated forever.
+A Refresh Token is a special kind of token that can be used to obtain a renewed Access Token. You are able to request new access tokens until the Refresh Token is blacklisted. It’s important that refresh tokens are [stored securely](/tokens/concepts/token-storage) by the application because they essentially allow a user to remain authenticated forever.
```

Comment:
A Refresh Token is a special kind of token used to obtain a renewed Access Token. You can request new access tokens until the Refresh Token is blacklisted. Applications must [store refresh tokens securely](/tokens/concepts/token-storage) because they essentially allow a user to remain authenticated forever.